scipy/optimize/_zeros_py.py

```diff
@@ -251,16 +251,16 @@ class of similar problems can be solved together.
     The above is the equivalent of solving for each value in ``(x, a)``
     separately in a for-loop, just faster:
 
-    >>> loop_res = [optimize.newton(f, x0, fprime=fder, args=(a0,))
+    >>> loop_res = [optimize.newton(f, x0, fprime=fder, args=(a0,),
+    ...                             maxiter=200)
```

Comment:
I guess if it is showing up in CI it doesn't hurt. FWIW, I did try to reproduce locally by running the before/after version of the test 1000 times with `pytest-repeat` (`pytest --count=1000 test.py`), and both cases passed every time for me on latest `master`:

```python
import numpy as np
import scipy
from scipy import optimize

def test_pr15022_original():
    f = lambda x, a: x**3 - a
    fder = lambda x, a: 3 * x**2
    rng = np.random.default_rng(1638083107694713882823079058616272161)
    x = rng.standard_normal(100)
    a = np.arange(-50, 50)
    loop_res = [optimize.newton(f, x0, fprime=fder, args=(a0,))
                for x0, a0 in zip(x, a)]

def test_pr15022_fixed():
    f = lambda x, a: x**3 - a
    fder = lambda x, a: 3 * x**2
    rng = np.random.default_rng(1638083107694713882823079058616272161)
    x = rng.standard_normal(100)
    a = np.arange(-50, 50)
    loop_res = [optimize.newton(f, x0, fprime=fder, args=(a0,),
                                maxiter=200)
                for x0, a0 in zip(x, a)]

```

`2000 passed in 86.38s (0:01:26)`

Perhaps that an external dependency that is different in CI vs. for me locally.

Reply:
I also could not reproduce. Thanks for trying!